mkdocs.yml

```diff
@@ -48,6 +48,7 @@ nav:
   - Home: index.md
   - Reference: reference.md
   - FAQS: faqs.md
+  - Contribution: contribution_guide.md
```

Comment:
@mosespace since we have moved this to the root....do you think maintaining this line is necessary